### PR TITLE
handle .R1.

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qp_woltka/tests/test_woltka.py [hunk 1]
```diff
@@ -470,7 +470,8 @@ def test_woltka_syndna_to_array(self):
             f'    bowtie2 -p 8 -x {database} -q '
             '${f} -S $PWD/sams/${sn}.sam --seed 42 --very-sensitive -k 16 '
             '--np 1 --mp "1,1" --rdg "0,1" --rfg "0,1" --score-min '
-            '"L,0,-0.05" --no-head --no-unal --un-gz $PWD/reads/${fn}\n',
+            '"L,0,-0.05" --no-head --no-unal --un '
```
Comment (charles-cowart) [inline qp_woltka/tests/test_woltka.py:473]: Just to confirm, it looks like (besides pathing) the switch also changed from --un-gz to just --un. Is this all right?

File: qp_woltka/woltka.py [hunk 2]
```diff
@@ -219,19 +249,34 @@ def woltka_to_array(files, output, database_bowtie2, prep, url, name):
...
-        'bowtie2 -p ${bt2_cores} '
-        f'-x {database_bowtie2} --interleaved - --seed 42 '
-        '--very-sensitive -k 16 --np 1 --mp "1,1" --rdg "0,1" --rfg "0,1" '
-        '--score-min "L,0,-0.05" --no-head --no-unal --no-exact-upfront '
-        "--no-1mm-upfront | cut -f1-9 | sed 's/$/\t*\t*/' | "
-        f'mxdx demux --file-map {files_list_fp} '
-        '--batch ${SLURM_ARRAY_TASK_ID} '
-        f'--batch-size {BATCHSIZE} --output-base {output}/alignments '
-        '--extension sam.xz')
+    if rev_exists:
```
Comment (charles-cowart) [inline qp_woltka/woltka.py:252]: The two bowtie2 lines seem very similar - is it possible to isolate the differences in them more?

File: qp_woltka/woltka.py [hunk 3]
```diff
@@ -402,7 +462,7 @@ def woltka_syndna_to_array(files, output, database_bowtie2, prep, url, name):
               '-q ${f} -S $PWD/sams/${sn}.sam ' +\
               '--seed 42 --very-sensitive -k 16 --np 1 --mp "1,1" ' + \
               '--rdg "0,1" --rfg "0,1" --score-min "L,0,-0.05" ' + \
-              '--no-head --no-unal --un-gz $PWD/reads/${fn}'
+              '--no-head --no-unal --un $PWD/reads/uneven/${fn/.gz/}'
```
Comment (charles-cowart) [inline qp_woltka/woltka.py:466]: Got it, it looks like --un-gz is intentionally changed to --un.

**Review by charles-cowart — CHANGES_REQUESTED** [review]
Everything looks good! Just one suggestion to make it easier to read.

File: qp_woltka/tests/test_woltka.py (hunk 1 above)
Comment (antgonza) [reply qp_woltka/tests/test_woltka.py:473]: yup

File: qp_woltka/woltka.py (hunk 2 above)
Comment (antgonza) [reply qp_woltka/woltka.py:252]: I personally prefer to have it like this as it's more clear to me ... basically having the full command there. However, if you feel strongly about this change, I can change it.

Comment (charles-cowart) [reply qp_woltka/woltka.py:252]: That's okay. I think if it's easier for you to read it this way it's better to keep it this way. Ty!

**Review by charles-cowart — APPROVED** [review]

**Pull request merged**